src/Agent.Worker/PluginInternalCommandExtension.cs

```diff
@@ -11,28 +11,25 @@
 
 namespace Microsoft.VisualStudio.Services.Agent.Worker
 {
-    public sealed class PluginInternalCommandExtension : AgentService, IWorkerCommandExtension
+    public class PluginInternalCommandExtension: BaseWorkerCommandExtension
```

Comment:
Are we using this as a base class somewhere else?

Reply:
nope, copy paste issue, I'll go back and make sure all of them are marked 'sealed'